src/lando/api/tests/conftest.py

```diff
@@ -432,6 +432,28 @@ def hg_clone(hg_server, tmpdir):
     return clone_dir
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    repo_dir = tmpdir.join("git_repo")
+    subprocess.run(["git", "init", repo_dir.strpath], check=True)
+    file = repo_dir.join("first")
+    file.write("first file!")
+    git_setup_user(repo_dir)
+    subprocess.run(["git", "add", file.basename], check=True, cwd=repo_dir)
+    subprocess.run(["git", "commit", "-m", "first commit"], check=True, cwd=repo_dir)
+    return repo_dir
+
+
+def git_setup_user(repo_dir):
```

Comment:
This should be a fixture. At least one that returns this method in its current form.